docs/devguide/flashfirmware.rst

```diff
@@ -1,58 +1,113 @@
 
-  yt up
+Clone the repository and change directory to it::
+
+  $ git clone <https://github.com/bbcmicrobit/micropython
 
-Start the build with either yotta::
+  $ cd bbcmicrobit/micropython
 
-  yt build
+Configure yotta to use the micro:bit target::
 
-...or use the Makefile::
+  yt target bbc-microbit-classic-gcc-nosd@https://github.com/lancaster-university/yotta-target-bbc-microbit-classic-gcc-nosd
+
+Run yotta update to fetch remote assets::
+
+  yt up
+
+Start the build using the makefile::
 
   make all
 
-The result is a microbit-micropython hex file (i.e. ``microbit-micropython.hex``)
-found in the build/bbc-microbit-classic-gcc-nosd/source from the root of the
-repository.
+The result is a microbit-micropython hex file
+(i.e. ``firmware.hex``) found in the
+build/bbc-microbit-classic-gcc-nosd/source from the root of the repository.
 
-The Makefile does some extra preprocessing of the source, which is needed only
-if you add new interned strings to ``qstrdefsport.h``. The Makefile also puts
+The Makefile does some extra preprocessing of the source, which is needed
+only if you add new interned strings to qstrdefsport.h. The Makefile also puts
 the resulting firmware at build/firmware.hex, and includes some convenience
 targets.
 
 Preparing firmware and a Python program
----------------------------------------
+=======================================
+
+Using ``tools/makecombinedhex.py`` you can combine the MicroPython firmware
+with a Python script and produce a hex file ready for uploading to the
+micro:bit.::
+
+  ./makecombinedhex.py <firmware.hex> <script.py> [-o <combined.hex>]
+  # Output goes to stdout if no filename is given.
+
+Using ``tools/hexlify.py`` you can turn a Python script into Intel HEX format
+to be concatenated at the end of the MicroPython firmware.hex.  A simple header
+is added to the script.::
+
+  ./hexlifyscript.py <script.py>
+  
+It also accepts data on standard input.
+
+
+micro:bit V2
+============
+
+This applies to MicroPython for the micro:bit V2, the source of which can be 
+found here: `microbit-foundation/micropython-microbit-v2 <https://github.com/microbit-foundation/micropython-microbit-v2>`_.
+
+The repository also contains a history of 
+`MicroPython firmware builds <https://github.com/microbit-foundation/micropython-microbit-v2/actions>`_.
+
+Dependencies
+------------
 
-tools/makecombined
+- `Arm gcc <https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads>`_
+- `gcc <http://gcc.gnu.org/install/>`_
+- `cMake <https://cmake.org/>`_
+- `git <https://git-scm.com/>`_ 
+- `ninja <https://ninja-build.org/>`_
+- `python <https://www.python.org/downloads/>`_
 
-hexlify
+Clone the repository and change directory to it::
 
+    $ git clone https://github.com/microbit-foundation/micropython-microbit-v2
+    $ cd micropython-microbit-v2
 
+Update the submodules::
+    $ git submodule update --init
```

Comment:
This didn't render correctly:
![image](https://user-images.githubusercontent.com/29712657/110932630-a3322a80-8323-11eb-9145-a6599483f125.png)

Please remember you can check the RtD PR build after each push.

Reply:
fixed in https://github.com/bbcmicrobit/micropython/pull/711/commits/939efde1f837cd9aea60e972e3bb135b215baff6